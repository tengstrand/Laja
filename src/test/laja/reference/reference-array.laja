
#set (x = [ 55, 66, 77] as array)
#set (y = [ 1, [ "c", "d" ] as array ] as array)

#write "target/test-output/reference-array.out"
x[1] = {x[1]}
y[1][1] = {y[1][1]}
#end
