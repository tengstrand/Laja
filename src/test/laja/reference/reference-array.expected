x[1] = 66
y[1][1] = d
